server/belga/search_providers.py

```diff
@@ -234,6 +235,8 @@ class Belga360ArchiveSearchProvider(superdesk.SearchProvider):
     def __init__(self, provider):
         super().__init__(provider)
         self.session = requests.Session()
+        with open('data/content_types.json') as f:
```

Comment:
can we just read this info from db?

Reply:
We use `content_types.json` to avoid mapping the wrong profile if there is two profile with the same label.
Belga instance exists some duplicate label due to new content profiles added to the json file, normally label should be unique. So user need to:
- Enable only one profile if there is multiple profile with the same label
- OR manually delete duplicate profile

What do you think?

Reply:
@idsvn , you can use `_id` from the DB, it's unique:

```
>>> db.getCollection('content_types').find({}, {_id: 1})

{
    "_id" : "alert"
}

/* 2 */
{
    "_id" : "brief"
}

/* 3 */
{
    "_id" : "tip"
}

/* 4 */
{
    "_id" : "text"
}

/* 5 */
{
    "_id" : ObjectId("5ecfa296b0be54a1722ae9cb")
}

/* 6 */
{
    "_id" : ObjectId("5efb4a9caec6c0fbdaf98905")
}
```